src/reach_study.cpp

```diff
@@ -92,8 +92,18 @@ void ReachStudy::run()
     const Eigen::Isometry3d& tgt_frame = target_poses_[i] * Eigen::AngleAxisd(M_PI, Eigen::Vector3d::UnitX());
 
     // Get the seed position
-    const std::vector<std::string> joint_names = ik_solver_->getJointNames();
-    std::map<std::string, double> seed_state = zip(joint_names, std::vector<double>(joint_names.size(), 0.0));
+    std::map<std::string, double> seed_state;
+    if (params_.seed_state.empty())
+    {
+      // no initialization give, use 0 positions
+      const std::vector<std::string> joint_names = ik_solver_->getJointNames();
+      seed_state = zip(joint_names, std::vector<double>(joint_names.size(), 0.0));
+    }
+    else
+    {
+      // use given initial positions
+      seed_state = params_.seed_state;
+    }
```

Comment:
Per the comment below, let's move this into the `runReachStudy` function and replace references in this function to `seed_state` with `params_.seed_state`